systray_unix.go

```diff
@@ -311,17 +312,28 @@ type tooltip = struct {
 	V3 string // description
 }
 
+// ScaleImagesTo may be set to automatically scale images to the given size.
+var ScaleImagesTo int = 32
```

Comment:
I'm not sure about the name - it does not provide context to what the value units are.
How about `TargetPixelWidth` or `ScaleSizePixels`?
Is there any way to avoid this global variable being used? it is inherently not thread safe, exposing as a method like the other global state would allow us to be safer.

Reply:
Can we default to a larger value? I prefer that we assume HiDPI now, which means being at least 2* this might be better.

Reply:
If you'd like to discuss then we could land the rest without this latest commit?

Reply:
Yes, you are right. It was a bad idea to include that in here. This is a separate topic. Removed and opening a new PR.